README.rst

```diff
@@ -37,6 +37,19 @@ or alternatively, install js-slang and run
   $ npm -g install js-slang   # Install js-slang
   $ js-slang [chapter] # default: 1
 
+Documentation
+-------------
+
+Source is documented here: https://sicp.comp.nus.edu.sg
+
+To build the documentation, run
+
+..code-block::
+
+  $ git clone https://github.com/source-academy/js-slang.git
+  $ cd js-slang
```

Comment:
```suggestion
  $ cd js-slang
  $ npm install # If additional permissions are required, run sudo npm install
```

I reproduced the steps and it seems like I needed to run `sudo npm install` before js-slang/doc/source is generated